man/docker-create.1.md

```diff
@@ -53,7 +53,7 @@ docker-create - Create a new container
 [**--read-only**[=*false*]]
 [**--restart**[=*RESTART*]]
 [**--security-opt**[=*[]*]]
-[**--stop-signal**[=*SIGNAL*]]
+[**--stop-signal**[=*SIGTERM*]]
```

Comment:
I think the original "SIGNAL" was correct here, these are not the "default" values